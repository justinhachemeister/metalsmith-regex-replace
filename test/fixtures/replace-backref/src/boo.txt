I saw one ghost talking to another ghost. The first ghost said "boo", then the
second ghost said "boooo". Then they both said "booooooooooooooo"!