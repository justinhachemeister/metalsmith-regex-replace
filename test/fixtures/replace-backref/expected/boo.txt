I saw one owl talking to another owl. The first owl said "hoo", then the
second owl said "hoooo". Then they both said "hooooooooooooooo"!